App/l10n/Resources/tr.lproj/BraveShared.strings

```diff
@@ -1213,6 +1216,72 @@
 /* See http://mzl.la/1G7uHo7 */
 "OpenPhoneSettingsActionTitle" = "Ayarları Aç";
 
+/* The description indicating the device is active # of days ago. The parameter substituted for \"%ld\" is the number of days. E.g.: Active 123 Days Ago */
+"opentabs.activePeriodDeviceDaysAgoTitle" = "%ld gün önce etkindi";
+
+/* The description indicating the device is active last time this month */
+"opentabs.activePeriodDeviceThisMonthTitle" = "Geçen Ay etkindi";
+
+/* The description indicating the device is active last time this week */
+"opentabs.activePeriodDeviceThisWeekTitle" = "Geçen Hafta etkindi";
+
+/* The description indicating the device is active last time today */
+"opentabs.activePeriodDeviceTodayTitle" = "Bugün Etkindi";
+
+/* The description indicating the device is active last time yesterday */
+"opentabs.activePeriodDeviceYesterdayTitle" = "Dün Etkindi";
+
+/* The description of the view showing no devices inside sync chain */
+"opentabs.noDevicesSyncChainPlaceholderViewDescription" = "Cihazlarınız ve etkin senkronizasyon türlerinizi görmek için bir senkronizasyon zincirine katılın";
```

Comment:
Edit: "`Cihazlarınızı` ve etkin senkronizasyon türlerinizi görmek için bir senkronizasyon zincirine katılın";